packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -267,7 +285,9 @@ export class SummarizerNode implements IRootSummarizerNode {
 
         this.refreshLatestSummaryCore(referenceSequenceNumber);
 
-        this._latestSummary = summaryNode;
+        if (summaryNode !== undefined) {
+            this._latestSummary = summaryNode;
+        }
```

Comment:
Interesting. So with the current code we'd set the latest summary to undefined if there's no pending summary?

Reply:
I think we still need it for the next summary as it will be updated before adding it to the pendingSummary.